abdulfatai360/banka
# #164984904 Add Admin Dashboard Template

#### What does this PR do?
Build out the templates of the admin dashboard

#### Description of Task to be completed?
- Have profile section of the admin
- Have section that detail the user account record of the bank

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto our local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Get a copy of the PR by typing  `git fetch origin pull/10/head:ft-add-admin-dashboard-164984904`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-add-admin-dashboard-164984904`
- Your directory will now be an exact copy of the PR.

#### What are the relevant pivotal tracker stories?
#164984904
